libdevice/imf/imf_fp64_dl.cpp

```diff
@@ -26,4 +26,21 @@ DEVICE_EXTERN_C_INLINE double __devicelib_imf_fmin(double a, double b) {
   return __fmin(a, b);
 }
 
+DEVICE_EXTERN_C_INLINE double __devicelib_imf_rcp64h(double x) {
+  uint64_t x_bits = __builtin_bit_cast(uint64_t, x);
```

Comment:
Could we use `sycl::bit_cast`?

Reply:
Hi, @steffenlarsen 
We can't use any sycl header files in libdevice since all libdevice files are built using the fresh built clang and sycl runtime/header files are not available at that time.
Thanks very much.

Reply:
I see, thank you! I suppose the host compiler shouldn't ever see this implementation anyway, right? In that case we should be able to assume the existence of the builtin from our device compiler.